multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -1,114 +1,83 @@
 from typing import List, Optional
 
-from .constants import BINARY, DATA, ENVIRONMENT, MODEL, MULTICLASS, OPTIMIZATION, REGRESSION
+from .constants import (
+    BEST_QUALITY,
+    BINARY,
+    DATA,
+    DEFAULT,
+    ENVIRONMENT,
+    HIGH_QUALITY_FAST_INFERENCE,
+    MEDIUM_QUALITY_FASTER_INFERENCE,
+    MODEL,
+    MULTICLASS,
+    OPTIMIZATION,
+    REGRESSION,
+)
 from .registry import Registry
 
 automm_presets = Registry("automm_presets")
 matcher_presets = Registry("matcher_presets")
 
 
 @automm_presets.register()
-def high_quality_fast_inference():
-    return {
+def default(presets: str = DEFAULT, hpo: bool = False):
```

Comment:
Why do we need `hpo` flag?

Reply:
This is to support the HPO presets, i.e., returning not only hyperparameters but also hyperparameter_tune_kwargs.